src/main/java/edu/harvard/iq/dataverse/ThumbnailServiceWrapper.java

```diff
@@ -46,6 +57,7 @@ public class ThumbnailServiceWrapper implements java.io.Serializable  {
     
     private Map<Long, String> dvobjectThumbnailsMap = new HashMap<>();
     private Map<Long, DvObject> dvobjectViewMap = new HashMap<>();
+    private Map<Long, Boolean> hasThumbMap = new HashMap<>();
 
     private String getAssignedDatasetImage(Dataset dataset, int size) {
```

Comment:
Any reason to keep this method around as it doesn't seem to be used anywhere?

Reply:
flyway updated, unused imports/method removed.